std::mem not imported in generated files

When regenerating webkit2gtk-rs with gir (master), the generated files fail to build because of a missing import:
```
   Compiling webkit2gtk v0.5.0 (/home/dvhaeren/projects/webkit2gtk-rs)
error[E0433]: failed to resolve: use of undeclared type or module `mem`
  --> src/auto/web_resource.rs:69:30
   |
69 |             let mut length = mem::uninitialized();
   |                              ^^^ use of undeclared type or module `mem`

error: aborting due to previous error

For more information about this error, try `rustc --explain E0433`.
error: Could not compile `webkit2gtk`.

To learn more, run the command again with --verbose.
```

--- a/src/analysis/functions.rs
+++ b/src/analysis/functions.rs
@@ -548,8 +548,13 @@ fn analyze_function(
             func.c_identifier.as_ref().unwrap_or(&func.name)
         );
         commented = true;
-    } else if !outs.is_empty() && !commented {
-        out_parameters::analyze_imports(env, func, imports);
+    } else if !commented {
+        if !outs.is_empty() {
+            out_parameters::analyze_imports(env, &func.parameters, func.version, imports);
+        }
+        if let Some(AsyncTrampoline{ref output_params, ..}) = trampoline {
+            out_parameters::analyze_imports(env, output_params, func.version, imports);
+        }
     }
 
     if async && !commented {

--- a/src/analysis/out_parameters.rs
+++ b/src/analysis/out_parameters.rs
@@ -12,6 +12,7 @@ use config;
 use env::Env;
 use library::*;
 use nameutil;
+use version::Version;
 
 #[derive(Clone, Copy, Debug, Eq, PartialEq)]
 pub enum Mode {
@@ -109,24 +110,27 @@ pub fn analyze(
     (info, unsupported_outs)
 }
 
-pub fn analyze_imports(env: &Env, func: &Function, imports: &mut Imports) {
-    for par in &func.parameters {
+pub fn analyze_imports(
+    env: &Env,
+    parameters: &[Parameter],
+    version: Option<Version>,
+    imports: &mut Imports,
+) {
+    for par in parameters {
         if par.direction == ParameterDirection::Out {
             match *env.library.type_(par.typ) {
-                Type::Bitfield(..) | Type::Enumeration(..) => imports.add("std::mem", func.version),
+                Type::Bitfield(..) | Type::Enumeration(..) => imports.add("std::mem", version),
                 Type::Fundamental(fund)
                     if fund != Fundamental::Utf8
                         && fund != Fundamental::OsString
                         && fund != Fundamental::Filename =>
                 {
-                    imports.add("std::mem", func.version)
+                    imports.add("std::mem", version)
                 }
-                _ if !par.caller_allocates => {
-                    match ConversionType::of(env, par.typ) {
-                        ConversionType::Direct | ConversionType::Scalar => (),
-                        _ => imports.add("std::ptr", func.version),
-                    }
-                }
+                _ if !par.caller_allocates => match ConversionType::of(env, par.typ) {
+                    ConversionType::Direct | ConversionType::Scalar => (),
+                    _ => imports.add("std::ptr", version),
+                },
                 _ => (),
             }
         }

